docs/source/author/content_style_guide.rst

```diff
@@ -0,0 +1,231 @@
+Preferably documents should be readable for students. 
+ 
+Any jargon used needs to be clearly explained. 
+
+
+Capitalisation Rules
+------------------------------------------------------------------------------
+
+In the majority of cases capitalisation should not be used for keywords and titles, with the following exceptions, where the phrase refers to a commonly used term that is often capitilised in the literature:
+
+- Computer Science
+- Computational Thinking
+- Digital Technologies (note that this is the correct form to refer to the subject area in NZ, with caps and plural; if it's referring to something other than the subject then use lower case e.g. "smartphones and other digital technologies", or even better, avoid the phrase e.g. "smartphones and other digital devices")
+- Sorting Network
+- Numeracy
+- Literacy
+ 
+The following wouldn't be capitalised:
+binary number(s), digits, binary digits
+ 
+
+
+Google Drive Folder Structure
+==============================================================================
+
+Including Images
+------------------------------------------------------------------------------
+
+All images included in a document have a comment which links to the original image file in a specified Google Drive Folder.
+
+
+Completed Documents
+------------------------------------------------------------------------------
+
+Once a document is completed:
+
+1. Prefix the document name with ``COMPLETE -``, e.g. ``Unit plan`` will become ``COMPLETE - Unit plan``.
+
+2. Put a note in the document header:
+
+    *A later version is under development, and will be released later in 2017.
+    No further updates will be made to this document.*
+
+3. Mark the document as ``Finished in Google Docs/Ready for GitHub`` on the Scorecard.
+
+3. Do not edit the document any further!
+
+
+Topic Folder
+------------------------------------------------------------------------------
+
+Each topic has its own folder in the ``Draft units and lessons`` folder, which contains all content relevant to that topic.
+A topic folder contains unit plan folders as they are in the scorecard.
+This applies to all topics even if they only contain one unit. 
+The structure of the topic folder, and folder and document names, are as they are below:
+ 
+.. code-block:: none
+
+  └── topic-name/
+      ├── Programming challenges/
+      ├── Resources/
+      ├── unit-name Unit/
+      │   ├── Lessons/
+      │   │   ├── 5-7/
+      │   │   ├── 8-10/
+      │   │   └── 11-14/
+      │   ├── Ideas
+      │   └── Unit plan
+      ├── Glossary
+      ├── Topic description
+      ├── Curriculum integration
+      └── Diff file
+
+Note:
+
+- If lessons do not cover all age groups then the relevant folder(s) should be omitted.
+
+- The ``Diff file`` is only required if the folder contains documents that are complete.
+
+
+Glossary
```

Comment:
We've discussed this a little in meetings but haven't made any decisions or worked on it much yet, so this will be something to come back to when there's time